internal/deploy/elastic_package.go

```diff
@@ -111,7 +110,8 @@ func checkElasticPackageProfile(ctx context.Context, kibanaProfile string) error
 		log.Trace("Not creating a new Elastic Package profile for " + kibanaProfile + ". Kibana config will be overriden")
 	}
 
-	elasticPackageProfileFile := filepath.Join(elasticPackageProfile, "stack", "kibana.config.yml")
+	// The kibana config file is only valid in 8.0.0, for other maintenance branches it's kibana.config.default.yml
```

Comment:
`"kibana.config.yml"` rather than `kibana.config.default.yml`? I see `kibana.config.default.yml` reference in L79

Reply:
In the config file we maintain for compose, there is only one file: `kibana.config.yml`. The elastic-package is the one having multiple config files. For that reason:

- in old L79:80 we had a comment and used the wrong file, that in new L79 we use the right file, the one we maintain.
- in new L114 we copy the previous file to the `.8x.yml` elastic-package file.

Not sure if I expressed myself correctly :S